pagopa/io-web-profile
# [IOPID-725] : add flag to remove spid test provider

## Short description

Add flag to remove spid test provider

## How to test

Change environment variable NEXT_PUBLIC_SPID_TEST_ENV_ENABLED to true/false and verify that test provider appear or disappear from the login page idp list
